docs/setup.md

```diff
@@ -0,0 +1,31 @@
+1. Download ngrok —  Go to ngrok.com/download and download for Mac OS X.
+    * Open Terminal type in:
+
+    ->cd Downloads
+    ->./ngrok http 5000
```

Comment:
I would prefer this to be in more generic language.  "In the terminal, go to the directory where `ngrok` is located and then type..."